anchore_engine/db/db_accounts.py

```diff
@@ -65,6 +70,10 @@ def update_state(name, new_state, session=None):
     if accnt.state == AccountStates.deleting or (accnt.state == AccountStates.enabled and new_state == AccountStates.deleting):
         raise InvalidStateError(accnt.state, new_state)
 
+    # Both Account Name and Type should be equal to "admin" for the Admin Account, but just to be safe...
+    if (accnt.name == 'admin' or accnt.type == 'admin') and new_state != AccountStates.enabled:
```

Comment:
There is a constant and an enum (AccountTypes.admin) for the admin account name and the admin account type respectively. Please use those so that any future updates are applied consistently.

See: https://github.com/anchore/anchore-engine/blob/master/anchore_engine/configuration/localconfig.py#L68 for the constant.

Reply:
Nice! Fixed.